.github/workflows/main.yml

```diff
@@ -0,0 +1,125 @@
+name: main
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*'
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-18.04
+
+    steps:
+      - run: |
+          echo "noop"
+
+  python-unit-tests:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        service: ["extractor", "transformer", "loader", "analyzer"]
+        python-version: [3.7]
+
+    # TODO(vmttn): Run tests in previously built container instead
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements-dev.txt
+      - run: python -m pytest -svv ${{ matrix.service }}/test
+        env:
+          PYROG_API_URL: http://test.com
+
+  golang-unit-tests:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        go-version: [1.14.x, 1.15.x]
```

Comment:
Can't we fix the version to the one we use to build the docker image?

Reply:
bumped the version in the docker image 🤫